app/views/shared/_landing_nav.html.erb

```diff
@@ -0,0 +1,34 @@
+<div id="landing_nav">
+  <div class="flex flex-row justify-between pt-10 px-28 w-full">
+    <a href="/">
+      <img src="<%= asset_path("icon/Logo.svg") %>" alt="DPI Logo" class="h-[2.5rem]">
+    </a>
+    <div class="bg-white opacity-70 flex items-center	justify-around border-slate-300 rounded-2xl w-[26rem] h-[3.2rem]">
+      <ul class="flex gap-10 text-s font-normal text-black max-w-[26]">
+        <li><a href="/" class="hover:text-gray-600 font-semibold">Home</a></li>
+        <li><a href="/" class="hover:text-gray-600">Demo</a></li>
+        <li><a href="/" class="hover:text-gray-600">Dashboard</a></li>
```

Comment:
Are you able to put the correct links?

Reply:
I'll update the Dashboard link but Demo page is just an idea.
We can decide what to put for later